documentation/sdk/sdk/00_sdk_overview.md

```diff
@@ -23,13 +23,9 @@ Happy hacking!
 2. To regenerate the documentation, run `npx jsdoc --configure jsdoc.json --verbose`
 
 ## Aleo Tools
-<!-- markdown-link-check-disable -->
 
-[Aleo SDK account generator](https://aleo.tools/)
+[Aleo SDK account generator](https://aleohq.github.io/aleo/)
```

Comment:
This should be fixed in a PR pointed at the sdk repo as these links will continue to break anytime the documentation is generated. I will open an issue to address this.

Reply:
@moAlobaidi can you open a PR to fix it directly?